db/structure.sql

```diff
@@ -1073,155 +1062,159 @@ CREATE UNIQUE INDEX unique_schema_migrations ON public.schema_migrations USING b
 -- PostgreSQL database dump complete
 --
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111123174941');
+INSERT INTO schema_migrations (version) VALUES ('20111123174941');
+
+INSERT INTO schema_migrations (version) VALUES ('20111123190839');
+
+INSERT INTO schema_migrations (version) VALUES ('20111124211843');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111123190839');
+INSERT INTO schema_migrations (version) VALUES ('20111128163114');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111124211843');
+INSERT INTO schema_migrations (version) VALUES ('20111128163317');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111128163114');
+INSERT INTO schema_migrations (version) VALUES ('20111128171949');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111128163317');
+INSERT INTO schema_migrations (version) VALUES ('20111130103929');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111128171949');
+INSERT INTO schema_migrations (version) VALUES ('20111130104648');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111130103929');
+INSERT INTO schema_migrations (version) VALUES ('20111130132854');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111130104648');
+INSERT INTO schema_migrations (version) VALUES ('20111210075847');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111130132854');
+INSERT INTO schema_migrations (version) VALUES ('20111210234133');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111210075847');
+INSERT INTO schema_migrations (version) VALUES ('20111210235123');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111210234133');
+INSERT INTO schema_migrations (version) VALUES ('20111210235206');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111210235123');
+INSERT INTO schema_migrations (version) VALUES ('20111210235226');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111210235206');
+INSERT INTO schema_migrations (version) VALUES ('20111210235259');
 
-INSERT INTO public.schema_migrations (version) VALUES ('20111210235226');
+INSERT INTO schema_migrations (version) VALUES ('20111213000431');
```

Comment:
@eitoball do you know how to fix this? @ZacharyCChang0828 and I tried today but dump just keeps providing `schema_migrations`

Reply:
No, I don't know. For now, I would recommend adding `pubilc.` prefix by hand. I know that it is not ideal way...

I will investigate `pg_dump` options. Some of options might solve this issue.

Reply:
I added public prefixes before schema_migrations.